bin/ruby-build

```diff
@@ -154,6 +154,10 @@ num_cpu_cores() {
   echo "${num:-2}"
 }
 
+install_package_direct() {
```

Comment:
I'm not a big fan of this change. `install_package_direct` is a misnomer when you're using your own custom mirror— it's not "direct" in that case.

I was fine with the default mirror being automatically skipped for `cache.ruby-lang.org` stuff. I don't think that the user should necessarily be conscious about this "magic" happening under the hood. All they need to know is that their downloads will stay fast by default.